### ISSUE TITLE
Hide assets

### ISSUE THREAD

**Issue by shg166** [sway2020/FindIt2#163]
Thank you for your help with the translation.

The beta version is able to hide assets by adding tags.
I don't know if it is possible to implement a way to add a tag to the asset description to hide it, but I have a suggestion.

When "&" is written in the description or asset name, the characters after "&" are not displayed In-game.
In the Asset Editor, the asset can be edited and confirmed without any problem, but only In-game, the characters are not displayed.
It is possible to describe it in the asset name, but it is better not to implement it because of the character limit.
Perhaps it could affect performance, as all the descriptions would be checked.

Can't we use this method to add "&" at the end of the description and use "hidden" to hide assets and "nosearch" to disable search?
1.How to describe the tags assumed in the description
`Example: end of description. & hidden nosearch`
2.If an identifier is needed as a prefix, add a symbol such as "#".
`Example: end of description. & #hidden #nosearch`
3.If CSV like description is preferred
`Example: end of description. & "hidden", "nosearch"`
4.In some cases, it would be possible for the creator to add arbitrary tags.
`Example: end of description. & jp japan`

If arbitrary tags can be implemented, it may be better to limit the number of tags. (10 at most?).
Would it be appropriate to optionally enable and disable this feature, or add a setting to the quick menu?
I don't know the design of the UI, so I'll leave it to you to figure out how to configure it.

This way, creators can explicitly hide or disable search for assets.
However, existing assets will need to be updated, as the description will need to be changed.
If not updated, the tag will be set using the hidden method added in the beta version.

**Comment by sway2020** [comment]
Yes this is possible to implement using some special keywords in asset description. I discussed this with some players and most people think this should be optional and disabled by default for transparency, because many people want to see all assets that are downloaded to their computer. I plan to add this feature in the future, not sure when.

**Comment by shg166** [comment]
Thanks for the reply.

I think the feature about hiding should be optional.
The default is to show all, and those who want to hide can enable it from the options.

Many Japanese creators in the Discord channel hosted by Mr.Kaminogi want to hide assets for parts.

I hope this will be implemented someday.

**Comment by kaminogi** [comment]
+1

I have made some networks that have lots of props should be hidden for UX. Some asset creators support this feature.

**Comment by sway2020** [comment]
Added in [5603c03](https://github.com/sway2020/FindIt2/commit/5603c03ec1cdca257317fc0c4d2f97ba12f22c7c). Will be included in the next beta update which will be pushed to the workshop in a few minutes.

How it works:

Asset creators will need to include the word creator_hidden or #creator_hidden in the asset description, and the mod user needs to enable an option in mod settings. I also added a new option in the extra filters panel which will always show those assets.
![20220805154329_1](https://user-images.githubusercontent.com/65274619/183220782-aef891ab-1dd2-49a0-a308-b85910cdffb6.jpg)
![20220805154418_1](https://user-images.githubusercontent.com/65274619/183220793-61044806-498c-416e-a054-36755b9a1934.jpg)
